api/cmd/transformer/.env.sample

```diff
@@ -1,6 +1,6 @@
 LOG_LEVEL=DEBUG
 
-STANDARD_TRANSFORMER_CONFIG={"transformerConfig":{"feast":[{"project":"default","entities":[{"name":"driver_id","valueType":"STRING","jsonPath":"$$.driver_id"}],"features":[{"name":"driver_statistics:avg_daily_trips_3","defaultValue":0},{"name":"driver_statistics:conv_rate","defaultValue":0}]}]}}
+STANDARD_TRANSFORMER_CONFIG={"transformerConfig":{"feast":[{"project":"default","entities":[{"name":"driver_id","valueType":"STRING","jsonPath":"driver_id"}],"features":[{"name":"driver_statistics:avg_daily_trips_3","valueType":"STRING","defaultValue":"0"},{"name":"driver_statistics:conv_rate","defaultValue":"0"}]}]}}
```

Comment:
Can you rebase this part

Reply:
Rebased and pushed

Reply:
Are you sure? it still remove `$$.` in jsonPath

Reply:
The json parser library that we used doesn't need `$.` prefix

Reply:
Ah, I didn't realize that you've changed `https://github.com/buger/jsonparser` to: `github.com/oliveagle/jsonpath`

Brb, updating the PR